cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml

```diff
@@ -0,0 +1,60 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+display_name: correct_test_task_type
```

Comment:
@HrishikeshGeedMS Could you please change these?

Reply:
Sure changing it